keyboards/lily58/keymaps/jhelvy/rules.mk

```diff
@@ -0,0 +1,31 @@
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+AUTO_SHIFT_ENABLE = yes     # Autoshift by holding down a key
+EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
+CONSOLE_ENABLE = no         # Console for debug(+400)
```

Comment:
Could you remove the parenthetical statements from these four lines? The numbers are rarely if ever accurate, so we're trying to get rid of them.